### DISCUSSION TITLE
[Feature] Browser default language

### DISCUSSION THREAD

**Discussion by waclaw66** [immich-app/immich#10720] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Immich should use browser defined language within `navigator.language` by default.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by MasterZhang007** [comment]
I agree, it annoyed me the language previously set was ALWAYS reset after upgrade to a new version, so I really really need a configurable environment varible to set default language rather than set it in user settings.

**Comment by bo0tzz** [comment]
#10849
